README.md

```diff
@@ -111,3 +111,34 @@ You now can use your custom Handlebars helper inside your templates as follows:
     {{!-- do stuff --}}
 {{/myDemo}}
 ```
+
+## Caching
+
+The rendering service offers the ability, to cache certain parts of templates. 
```

Comment:
the "," is not needed here. "certain parts" feels weird. why not "... the ability to cache the parsed template between requests for faster rendering."